source/Detector/Amptek/SGM/AmptekSDD123EPICSDetectorManager.h

```diff
@@ -63,11 +63,11 @@ protected slots:
 
 protected:
 	/// helper function to handle continuous spectrum data update
-	void dataHelper(AMDSDataHolder *spectrum, int count, double elapsedTime);
+	void dataHelper(AMDSDataHolder *spectrum, double count, double elapsedTime);
```

Comment:
Isn't count the number of data positions?  That should definitely be an int if that's the case.